src/components/Header.jsx

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+export default function Header() {
+  return (
+    <header className="header">
+      <Link href="/" aria-label="To home page" className="header__link">
+        <img className="header__logo" src="/sparkbox-logo-text.svg" alt="Sparkbox logo" />
```

Comment:
This link isn't working because it doesn't have the `<a>` as a child element.

```suggestion
        <a href="replace">
          <img
            className="header__logo"
            src="/sparkbox-logo-text.svg"
            alt="Sparkbox logo"
          />
        </a>
```